File: PYTHON/ejercicio55.py
numeros = []
suma = 0

for i in range(5):
    n = int(input("Número: "))
    numeros.append(n)
    suma += n

mayor = numeros[0]
menor = numeros[0]

for n in numeros:
    if n > mayor:
        mayor = n

    if n < menor:
        menor = n

print("Mayor:", mayor)
print("Menor:", menor)
print("Promedio:", suma / len(numeros))

File: JS/ejercicio55.js
let numeros = [10, 5, 20, 8, 15];

let mayor = numeros[0];
let menor = numeros[0];
let suma = 0;

for (let i = 0; i < numeros.length; i++) {
    if (numeros[i] > mayor) {
        mayor = numeros[i];
    }

    if (numeros[i] < menor) {
        menor = numeros[i];
    }

    suma += numeros[i];
}

let promedio = suma / numeros.length;

console.log("Mayor:", mayor);
console.log("Menor:", menor);
console.log("Promedio:", promedio);